### PR TITLE
openshot-qt: unbreak

### PR DESCRIPTION
**Pull request by B4dM4n**
###### Motivation for this change

Unbreak `openshot-qt`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by B4dM4n** [comment]
Result of `nixpkgs-review pr 108689` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>libsForQt5.libopenshot</li>
    <li>libsForQt512.libopenshot</li>
    <li>libsForQt514.libopenshot</li>
    <li>openshot-qt</li>
  </ul>
</details>

File: pkgs/applications/video/openshot-qt/libopenshot.nix [hunk 1]
```diff
@@ -19,7 +19,14 @@ stdenv.mkDerivation rec {
...
 
-  patchPhase = ''
+  patches = [
+    (fetchpatch {
+      name = "fix-build-with-gcc-10.patch";
+      url = "https://github.com/OpenShot/libopenshot/commit/13290364e7bea54164ab83d973951f2898ad9e23.diff";
+      sha256 = "0i7rpdsr8y9dphil8yq75qbh20vfqjc2hp5ahv0ws58z9wj6ngnz";
+    })
+  ];
+  postPatch = ''
```
Comment (SuperSandro2000) [inline pkgs/applications/video/openshot-qt/libopenshot.nix:29]:
```suggestion
  ];

  postPatch = ''
```

Comment (B4dM4n) [reply pkgs/applications/video/openshot-qt/libopenshot.nix]: Applied

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108689` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>libsForQt5.libopenshot (libsForQt515.libopenshot)</li>
<li>libsForQt512.libopenshot</li>
<li>libsForQt514.libopenshot</li>
<li>openshot-qt</li>
</ul>
</details>

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Squash the commits.

**Pull request merged**